runtime/lua/vim/lsp.lua

```diff
@@ -232,6 +232,12 @@ local function validate_client_config(config)
     flags           = { config.flags, "t", true };
     get_language_id = { config.get_language_id, "f", true };
   }
+  assert(
+    (not config.flags
+      or not config.flags.debounce_did_change_notify
+      or type(config.flags.debounce_did_change_notify) == 'number'),
```

Comment:
I'm not sure how I feel about debounce_did_change_notify, I know what it is, but maybe debounce_text_changes? Idk this is nitpicky

Reply:
`debounce_text_changes` also sounds good to me :+1: 
I also wasn't very convinced about `debounce_did_change_notify`

Reply:
Pushed the change.